file_sync_only/Tiltfile

```diff
@@ -0,0 +1,43 @@
+# -*- mode: Python -*-
+
+# module function
+# non build development with only file sync
+def file_sync_only(image='', manifests=[], live_update=[]):
+    if type(manifests) == "string":
+        manifests = [manifests]
+
+    # ex) image is "nginx" or "nginx:1.17" format
+    # "nginx" => return "nginx" and "1.16" (from manifests)
+    # "nginx:1.17" => return "nginx" and "1.17"
+    deployed_image, deployed_tag = _get_current_tag(image, manifests)
+
+    # ":" is null command 
+    custom_build(deployed_image, ':',
+        tag=deployed_tag,
+        deps=["."],
+        skips_local_docker=True,
+        live_update=live_update
+    )
+    k8s_yaml(manifests)
+
+
+# get current tags from image setting from first arg in file_sync_only() or in manifests
+def _get_current_tag(image='', manifests=[]):
+    p = image.split(':')
+    if len(p) == 2:
+        return p[0], p[1]
+
+    tags = []
+
+    for m in manifests:
+        output = str(local(r"grep --include '*.yaml' --include '*.yml' 'image: %s' %s | cut -d ':' -f 3 || true" % (image, m))).strip()
```

Comment:
i don't think you need the `--include`? you're already passing the file name in directly, right?

Reply:
@nicks Thanks a lot! I fix it  and I add `uniq` for same tags.
https://github.com/tilt-dev/tilt-extensions/pull/77/commits/4d987862b4f5b27c8f5aba4947ad23df5a3ca6f2#diff-fd6c7cf6851d50036c7539c1ee12e7f9R33